learn/get-started/create-account.md

```diff
@@ -66,7 +66,7 @@ func main() {
 </example>
 
 [TODO: should this only show if viewing the SDK examples?]
-You might notice that, in the SDK, you ask for the *account ID* instead of the public key. That’s because an account’s ID *is* its public key.
+You might notice that, in the SDK, you are asked for the *account ID* instead of the public key. That’s because an account’s ID *is* its public key.
```

Comment:
Hmmm, I think this definitely should have been “ask” instead of “are asked”—at least, I was trying to refer to the fact that you call `pair.accountId()` or `pair.getAccountId()` rather than what one might expect from a key pair, e.g. `pair.publicKey()`.

Would if be clearer if it was:

> You might notice that, in the SDK, you call `pair.accountId()` instead of something like `pair.publicKey()`. That’s because an account’s ID *is* its public key.